### ISSUE TITLE
bug: eza should not show the `-> linked-to` part in oneline mode

### ISSUE THREAD

**Issue by moonfruit** [eza-community/eza#662]
If eza does something unexpected, or its output looks wrong, or it displays an error on the screen, or if it outright crashes, then please include the following information in your report:

- The version of eza being used (`eza --version`)
	<img width="371" alt="图片" src="https://github.com/eza-community/eza/assets/1070090/17cd4361-b15f-4944-8336-f846d9d610bc">

- The command-line arguments you are using
	`eza -1`

- Your operating system and hardware platform
  <img width="252" alt="图片" src="https://github.com/eza-community/eza/assets/1070090/8f6eef3d-258d-40bb-84d4-7cffd0c2d3d7">

If it’s a crash, please include the full text of the crash that gets printed to the screen. If you’re seeing unexpected behaviour, a screenshot of the issue will help a lot.

---

I think eza should not show the `-> linked-to` part in oneline mode.

Now eza output:
<img width="886" alt="图片" src="https://github.com/eza-community/eza/assets/1070090/97185a5d-6a2b-4383-ae31-30190e51baa4">

As `ls -1` shows:
<img width="246" alt="图片" src="https://github.com/eza-community/eza/assets/1070090/5f2186a0-88f4-4dff-ac51-c0c03ffb02ce">

**Comment by holmesrichards** [comment]
It does the same in piped output with or without the -1 option, which seems wrong to me:

```
$ eza | bat -p
'v -> /home/rsholmes/quotes/x y/z'/u
'x y'
'xyc -> /home/rsholmes/quotes/x y'/c
xyl -> /home/rsholmes/quotes/'x y'
'xyzab -> /home/rsholmes/quotes/x y/z'/'a b'
'xyzijk -> /home/rsholmes/quotes/x y/z/i/j'/k
'xyzl -> /home/rsholmes/quotes/x y'/z
'xyzpqr -> /home/rsholmes/quotes/x y/z/p q'/r
'xyzpqst -> /home/rsholmes/quotes/x y/z/p q'/'s t'
```
ls doesn't:
```
$ ls | bat -p
v
x y
xyc
xyl
xyzab
xyzijk
xyzl
xyzpqr
xyzpqst
```

**Comment by MartinFillon** [comment]
Well I dont know if this is really an error, as we dont mimic the result of ls but I can ask for maintainers they thoughts

**Comment by holmesrichards** [comment]
I would argue that at least in piped output, showing the link is undesired behavior.

**Comment by MartinFillon** [comment]
Yeah I can understand that argument tho, and @holmesrichards if you want to talk more with the team join us on element (see readme) you seem to be someone that finds a lot of bugs (thanks) so we can use your help

**Comment by moonfruit** [comment]
If `eza` don't want to fix this, then at the very least `eza` should provide an option to turn off the display of the `-> linked-to` part.
Please consider this feature request, @MartinFillon
